<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>838</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="PacketsTableWidget" name="tableWidget">
        <property name="styleSheet">
         <string notr="true">color: black;
background-color: white;
selection-color: yellow;
selection-background-color: grey;

border-style: outset;
border-width: 2px;
border-color: beige;</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>12</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Timestamp</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Packet Size</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 2 Source</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 2 Destination</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 3 Source</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 3 Destination</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 4 Source</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Layer 4 Destination</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Additional Information</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_Selection"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_Selection">
   <property name="text">
    <string>Save Selection</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PacketsTableWidget</class>
   <extends>QTableWidget</extends>
   <header>packetstablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
